integration_tests/src/main/python/orc_write_test.py

```diff
@@ -360,6 +360,23 @@ def create_empty_df(spark, path):
         conf={'spark.rapids.sql.format.orc.write.enabled': True})
 
 
+hold_gpu_configs = [True, False]
+@pytest.mark.parametrize('hold_gpu', hold_gpu_configs, ids=idfn)
+def test_async_writer(spark_tmp_path, hold_gpu):
+    data_path = spark_tmp_path + '/ORC_DATA'
+    num_rows = 2048
+    num_cols = 10
+    orc_gen = [int_gen for _ in range(num_cols)]
+    gen_list = [('_c' + str(i), gen) for i, gen in enumerate(orc_gen)]
+    assert_gpu_and_cpu_writes_are_equal_collect(
+        lambda spark, path: gen_df(spark, gen_list, length=num_rows).coalesce(1).write.orc(path),
```

Comment:
why `coalesce(1)`?

Reply:
Sharp eye. 

My guess is that `coalesce(1)` isn't strictly required for the test to pass.

Reply:
Oh good catch. I think I just copy-pasted without looking. I copy-pasted this from `test_async_writer()` in `parquet_write_test.py`, which in turn was copied from another test in the same file. I fixed both async writer tests.